frontend/public/components/utils/alerts.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { Alert, AlertActionCloseButton, List, ListItem } from '@patternfly/react-core';
+
+export const ExpandableAlert: React.FC<CustomAlertProps> = ({alerts, variant}) => {
+  const alertCount = alerts.length;
+  const [visible, setVisible] = React.useState(true);
+  const [expanded, setExpanded] = React.useState(false);
+
+  const alertContent = alertCount > 1 ? <List>{_.map(alerts, (error, i) => <ListItem key={i}>{error}</ListItem>)}</List> : alerts;
+
+  if (visible) {
+    return <Alert
+      isInline
+      variant={variant}
+      className="co-expandable-alert"
+      title={<React.Fragment>{`There are ${alertCount} ${variant} alerts.`}<button type="button" className="btn btn-link" onClick={() => setExpanded(!expanded)}>{expanded ? 'Hide' : 'Show'} Details</button></React.Fragment>}
+      action={<AlertActionCloseButton onClose={() => setVisible(!visible)} />}
```

Comment:
Do we want the close button? I don't see a lot of value in it since it will just reappear next time you visit the page.

We should try to be consistent here with other alerts. I don't see any close buttons in #1763.